mobile/library/common/common/default_system_helper.cc

```diff
@@ -4,4 +4,16 @@ namespace Envoy {
 
 bool DefaultSystemHelper::isCleartextPermitted(absl::string_view /*hostname*/) { return true; }
 
+envoy_cert_validation_result
+DefaultSystemHelper::validateCertificateChain(const std::vector<std::string>& /*certs*/,
+                                              absl::string_view /*hostname*/) {
+  envoy_cert_validation_result result;
+  result.result = ENVOY_FAILURE;
+  result.tls_alert = 80; // internal error
+  result.error_details = "Certifcate verification not implemented ";
```

Comment:
nit: s/Certifcate/Certificate and '.' instead of whitespace at the end.

Reply:
Doh! Done.